stubs/chevron/chevron/renderer.pyi

```diff
@@ -0,0 +1,19 @@
+from _typeshed import StrOrBytesPath, SupportsRead
+from typing import Any, Sequence
+
+from tokenizer import tokenize as tokenize
+
+g_token_cache: dict[str, list[tuple[str, str]]]  # undocumented
+
+def render(
+    template: SupportsRead[str] | str = ...,
+    data: dict[str, Any] = ...,
+    partials_path: StrOrBytesPath | None = ...,
```

Comment:
It calls `os.path.join(partials_path, name + path_ext)`, where `path_ext` is a string and so `name + path_ext` is also  a string. So bytes paths won't work.